src/Runner.js

```diff
@@ -40,7 +42,7 @@ export default class Runner extends EventEmitter {
     this._createProcess = (options && options.createProcess) || createProcess;
     this.options = options || {};
     this.workspace = workspace;
-    this.outputPath = tmpdir() + '/jest_runner.json';
+    this.outputPath = `${tmpdir()}/jest_runner_${this.workspace.outputFileSuffix || ''}.json`;
```

Comment:
Should we use a platform-agnostic path separator here?

Reply:
good point, I wonder why it didn't break on windows before? oh well, we should do the right thing regardless...